ray-operator/controllers/ray/metrics/ray_cluster_metrics.go

```diff
@@ -47,6 +50,12 @@ func NewRayClusterMetricsManager(ctx context.Context, client client.Client) *Ray
 			[]string{"name", "namespace", "owner_kind"},
 			nil,
 		),
+		rayClusterProvisionedReady: prometheus.NewDesc(
+			"kuberay_cluster_condition_provisioned",
+			"Indicates whether the RayCluster is provisioned and ready",
```

Comment:
```suggestion
			"Indicates whether the RayCluster is provisioned",
```
defer to @kevin85421 for clearer description

Reply:
"Indicates whether the RayCluster is provisioned" is more precise. A provisioned cluster isn't necessarily ready.

```go
// RayClusterProvisioned indicates whether all Ray Pods are ready for the first time.
// After RayClusterProvisioned is set to true for the first time, it will not change anymore.
RayClusterProvisioned RayClusterConditionType = "RayClusterProvisioned"
```

Reply:
Fixed!